.npmrc

```
@@ -0,0 +1,2 @@
+npm_config_save_exact=true
```

I think this should be `save-exact` like the webapp and mobile repos
```suggestion
save-exact=true
```

---

done.